### DISCUSSION TITLE
Consultar estado de una factura

### DISCUSSION THREAD

**Discussion by goear** [Barnetik/tbai-php-lib#49] [Q&A]
Kaixo @arraintxo y compañía, sabéis si es posible consultar la información de una factura entregada en ticketbai?

**Comment by arraintxo** [comment]
Cada una de las administraciones tiene la opción de mirar el registro de facturas presentadas. 
En Bizkaia a través del LROE, en Gipuzkoa a través de TicketBai Ataria y en Araba hay una sección para consultar las facturas enviadas también.

No sé si es lo a lo que te refieres o si querías consultar las facturas presentadas usando la librería. Esto no está implementado y no sé hasta que punto es posible.

**Reply by goear** [reply]
Kaixo @arraintxo si, preguntaba a através de API, porque vi esto

https://ticketbaiws.eus/es/documentacion-api/ticketbai-get/

y pensaba que las administraciones no daban esa posibilidad y claro no se si estos registran los envios en su propio sistema y consutan a su base de datos o es que realmente está disponible ese endpoint por parte de las haciendas.

**Reply by goear** [reply]
desde Gipuzkoa me han dicho que está disponible para entorno de pruebas:

https://www.gipuzkoa.eus/es/web/ogasuna/-/ticketbai-kontsulta-nuevo-servicio-rest

a ver que me responden las otras 2 administraciones.

**Reply by arraintxo** [reply]
Supongo que en Bizkaia será posible acceder a través del LROE y Araba irá por el mismo camino que Gipuzkoa igual que han hecho hasta ahora. 

En cuanto a la librería, no tenemos planes de añadir esta funcionalidad en el corto plazo. Sin embargo, estamos abiertos a colaboraciones de la comunidad, ya sea mediante aportaciones de código o patrocinios que permitan priorizar y desarrollar nuevas funcionalidades. 

Aún así seguimos interesados en saber que te responden desde Araba y Bizkaia.

Eskerrik asko.
